.github/workflows/flutter_build_android_melos.yaml

```diff
@@ -0,0 +1,58 @@
+name: Deploy Android
+on:
+  workflow_call:
+    inputs:
+      app-dir:
+        required: true
+        type: string
+      environment: # options development|staging|production
+        required: true
+        type: string
+      packageName: # this should be optional.. https://github.com/r0adkll/upload-google-play/issues/103
+        required: true
+        type: string
+
+    secrets:
+      android_key_asc:
+        required: true
+      android_key_passphrase:
+        required: true
+      android_keystore_asc:
+        required: true
+      android_keystore_passphrase:
+        required: true
+      play_store_gservice_account:
+        required: true
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin' # See 'Supported distributions' for available optionsjava-version
+          java-version: '11'
```

Comment:
Zelfde hier, is het verstandig om hier de versie te pinnen?

Reply:
Wellicht is er een manier om de laatste LTS release te automatiseren. Maar 11 is de laatste voor nu.

Reply:
Even opgezocht. Het kan in ieder geval niet met actions/setup-java@v3
```
The java-version input supports an exact version or a version range using [SemVer](https://semver.org/) notation:

major versions: 8, 11, 16, 17
more specific versions: 17.0, 11.0, 11.0.4, 8.0.232, 8.0.282+8
early access (EA) versions: 15-ea, 15.0.0-ea, 15.0.0-ea.2, 15.0.0+2-ea
```

Reply:
17 is overigens de echte laatste LTS release, alleen daar kreeg ik gradle issues op.